oak_restricted_kernel/src/lib.rs

```diff
@@ -341,58 +341,60 @@ pub fn start_kernel(info: &BootParams) -> ! {
         sev_status,
     );
 
-    let application: payload::Application = {
+    let application_bytes: Box<[u8]> = {
```

Comment:
`Box<[u8]>`? That seems a bit of an odd type -- why not `Vec<u8>`?

Reply:
It's what the various functions and structs in `oak_restricted_kernel/src/payload.rs` expect. Unsure what the original rational was, but decided to just keep the type signature for now. :)